takoyaki/src/main/java/frc/robot/subsystems/Climb/FalconBrakeModeClimb.java

```diff
@@ -14,145 +13,96 @@
-      }
-    }
-    /*
-     * else {
-     * if (goingUp) {
-     * if (left.getSelectedSensorPosition() <= Constants.kClimb.TOP_ENCODER_VAL) {
-     * left.set(ControlMode.PercentOutput, 0);
-     * }
-     * if (right.getSelectedSensorPosition() <= Constants.kClimb.TOP_ENCODER_VAL) {
-     * right.set(ControlMode.PercentOutput, 0);
-     * }
-     * if (left.getSelectedSensorPosition() <= Constants.kClimb.TOP_ENCODER_VAL
-     * && right.getSelectedSensorPosition() <= Constants.kClimb.TOP_ENCODER_VAL) {
-     * closedLoop = false;
-     * }
-     * } else {
-     * if (left.getSelectedSensorPosition() >= Constants.kClimb.BOTTOM_ENCODER_VAL)
-     * {
-     * left.set(ControlMode.PercentOutput, 0);
-     * }
-     * if (right.getSelectedSensorPosition() >= Constants.kClimb.BOTTOM_ENCODER_VAL)
-     * {
-     * right.set(ControlMode.PercentOutput, 0);
-     * }
-     * if (left.getSelectedSensorPosition() >= Constants.kClimb.TOP_ENCODER_VAL
-     * && right.getSelectedSensorPosition() >= Constants.kClimb.TOP_ENCODER_VAL) {
-     * closedLoop = false;
-     * }
-     * }
-     * 
-     * }
-     */
-
+  public void setSetpoint(double setpoint) {
+    this.setpoint = setpoint;
   }
 
-  public double normalizeInput(double joystickInput) {
-    if (joystickInput > 0.5) {
-      return Constants.kClimb.OPEN_LOOP_UP_POWER;
-    } else if (joystickInput < -0.5) {
-      return Constants.kClimb.OPEN_LOOP_DOWN_POWER;
-    }
-    return 0;
+  public void extendClimb() {
+    this.setpoint = Constants.kClimb.TOP_ENCODER_VAL;
   }
 
-  @Override
-  public void simulationPeriodic() {
+  public void retractClimb() {
+    this.setpoint = Constants.kClimb.BOTTOM_ENCODER_VAL;
   }
 
   public void stopClimb() {
     closedLoop = false;
     left.set(ControlMode.PercentOutput, 0);
-    right.set(ControlMode.PercentOutput, 0);
   }
 
-  public void extendClimb() {
-    closedLoop = true;
-    goingUp = true;
-    left.set(ControlMode.PercentOutput, Constants.kClimb.CLOSED_LOOP_UP_POWER);
-    right.set(ControlMode.PercentOutput, Constants.kClimb.CLOSED_LOOP_UP_POWER);
+  public void runClimb() {
+    closedLoop = false;
+    left.set(ControlMode.PercentOutput, Constants.kClimb.OPEN_LOOP_UP_POWER);
   }
 
-  public void retractClimb() {
-    closedLoop = true;
-    goingUp = false;
-    left.set(ControlMode.PercentOutput, Constants.kClimb.CLOSED_LOOP_DOWN_POWER);
-    right.set(ControlMode.PercentOutput, Constants.kClimb.CLOSED_LOOP_DOWN_POWER);
+  public void reverseClimb() {
+    closedLoop = false;
+    left.set(ControlMode.PercentOutput, Constants.kClimb.OPEN_LOOP_DOWN_POWER);
   }
 
-  private void motorConfig(WPI_TalonFX motor) {
-    motor.configFactoryDefault();
-    motor.setSelectedSensorPosition(0);
-    motor.setNeutralMode(NeutralMode.Brake);
-    motor.configOpenloopRamp(Constants.kClimb.OPEN_LOOP_RAMP_RATE);
+  public void releasePassiveHook() {
+    this.setSetpoint(Constants.kClimb.TOP_ENCODER_VAL - Constants.kClimb.UNHOOK_DISTANCE);
```

Comment:
this is good to have but also im not sure if we will use it, we can release the passive hook by just pulling up all the way i believe

Reply:
We can't pull up all the way or we go above high bar and get a penalty. Once we are hanging from traversal we just want to go up enough to release the passive hook